ql/qldefines.hpp.cfg

```diff
@@ -0,0 +1,228 @@
+/* -*- mode: c++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
```

Comment:
With an in-source build, this modifies the original and leaves the repo dirty, but I guess it can't be helped — and the common practice with cmake is to use out-of-source builds, right?  Come to think of it, I'm not sure we're doing that in the CI workflow.  I'll have to fix that.

Reply:
@luigo out of source builds are the _beauty_ of it.  I would indeed change the workflow (which does a lot of in-source `cmake .`) I can allow you to change that local branch, sure.